accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableutils/impl/AuthorizableCreatorServiceImpl.java

```diff
@@ -123,21 +124,20 @@ private void installAuthorizableConfigurationBean(final Session session,
         else {
 
             // update name for both groups and users
-            setAuthorizableName(authorizableForPrincipalId, vf, authorizableConfigBean.getPrincipalName());
+            setAuthorizableName(authorizableForPrincipalId, vf, authorizableConfigBean);
+            // update password for users
+            if (!authorizableForPrincipalId.isGroup() && !authorizableConfigBean.isSystemUser()
+                    && StringUtils.isNotBlank(authorizableConfigBean.getPassword())) {
+                ((User) authorizableForPrincipalId).changePassword(authorizableConfigBean.getPassword());
+            }
 
-            if (authorizableConfigBean.isGroup()) {
+            // move authorizable if path changed (retaining existing members)
+            handleIntermediatePath(session, authorizableConfigBean, history,
+                    authorizableInstallationHistory, userManager);
 
-                handleIntermediatePath(session, authorizableConfigBean, history,
-                        authorizableInstallationHistory, userManager);
-                mergeGroup(history, authorizableInstallationHistory,
-                        authorizableConfigBean, userManager);
+            mergeGroup(history, authorizableInstallationHistory,
+                    authorizableConfigBean, userManager);
 
-            } else {
-                String msg = "- authorizable " + principalId
```

Comment:
This message is removed now - users can be moved now (and references to it from other groups are maintained).